src/citrine/resources/ingestion.py

```diff
@@ -339,17 +467,13 @@ def build_from_file_links(self,
             response = self.session.post_resource(path=self._get_path(), json=req)
         except BadRequest as e:
             if e.api_error is not None:
-                errors = e.api_error.validation_errors
-                if len(errors) == 0:
-                    errors = [ValidationError.build({"failure_message": e.api_error.message,
-                                                     "failure_id": "failure_id"})
-                              ]
+                if len(e.api_error.validation_errors) != 0:
```

Comment:
Nitpick, since it's come up before: "For sequences, (strings, lists, tuples), use the fact that empty sequences are false:"

https://peps.python.org/pep-0008/

Reply:
Okay, I recalled that PEP completely backwards.  Weird.